### PR TITLE
add bio benchmarks summary plot

### PR DESCRIPTION
**Pull request by TorkelE**
Adds summary plots for the 5 bio benchmarks (to be used in Catalyst paper).

### CODE REVIEW

File: benchmarks/Bio/BCR.jmd [hunk 1]
```diff
@@ -34,13 +34,13 @@ obs = [eq.lhs for eq in observed(rn)]
 show(to)
 
 tspan = (0.,tf)
-@timeit to "ODEProb No Jac" oprob = ODEProblem{true, SciMLBase.FullSpecialize}(osys, Float64[], tspan, Float64[])
```
Comment (ChrisRackauckas) [inline benchmarks/Bio/BCR.jmd:37]: No, we should keep this.

File: benchmarks/Bio/egfr_net.jmd [hunk 2]
```diff
@@ -34,12 +34,12 @@ obs = [eq.lhs for eq in observed(rn)]
 show(to) 
 
 tspan = (0.,tf)
-@timeit to "ODEProb No Jac" oprob = ODEProblem{true, SciMLBase.FullSpecialize}(osys, Float64[], tspan, Float64[])
```
Comment (ChrisRackauckas) [inline benchmarks/Bio/egfr_net.jmd:37]: why are these removed in each?

**Comment by TorkelE** [comment]
Not sure, I only ever added a bit at the end and changed nothing else. My fork must never have contained the `{true, SciMLBase.FullSpecialize}` in the first place. Not sure why because I just re-forked it to ensure it was new. Will fix.
